internal/deploy/elastic_package.go

```diff
@@ -111,7 +110,8 @@ func checkElasticPackageProfile(ctx context.Context, kibanaProfile string) error
 		log.Trace("Not creating a new Elastic Package profile for " + kibanaProfile + ". Kibana config will be overriden")
 	}
 
-	elasticPackageProfileFile := filepath.Join(elasticPackageProfile, "stack", "kibana.config.yml")
+	// The kibana config file is only valid in 8.0.0, for other maintenance branches it's kibana.config.default.yml
+	elasticPackageProfileFile := filepath.Join(elasticPackageProfile, "stack", "kibana.config.8x.yml")
```

Comment:
@mtojek the `8x` file name is valid for the 8.0 branch of the stack?

Reply:
This isn't meant to be branch or specific stack version like 8.0.0, but in general the stack family 8x.

Reply:
I asked because for our 7.15 and 7.16 branches, we are using here `kibana.config.default.yml` file, but in master and 8.0, we are going to use `kibana.config.8x.yml`, to follow elastic-package's behaviour